src/environments/environment.template.ts

```diff
@@ -37,6 +37,7 @@ export const environment = {
       defaultName: 'REPUX',
       precision: 18,
       format: '1.2-18',
+      shortFormat: '1.2-4',
```

Comment:
I think that truncating prices is bad idea. When real price is e.g. 0,00001 then we display 0,0000. If you want to truncate price, you should first check if you not changing value.

Reply:
Value is rounded - as per request from Ola :)